### ISSUE TITLE
Color customization for top-bar-section active item

### ISSUE THREAD

**Issue by SukramD** [sympa-community/sympa#146]
Hi,

I had the problem that I was unable to customize the color of the active element in the top bar with the normal configuration tools:
![sympa_layout](https://user-images.githubusercontent.com/34739219/34251432-5c050390-e640-11e7-9e39-0d69284a897f.png)
Basically, the blue item should have been orange like it is on the submenu.
To solve this I had to change the following entry in the foundation.css:
```
    .top-bar-section li.active:not(.has-form) a:not(.button) {
      background: #FF6B12;
      color: #FFFFFF;
      line-height: 2.8125rem;
      padding: 0 0.9375rem; }
```
Would be great if this can be configured with the normal configuration tools.
This happened with sympa version 6.2.22

**Comment by aepli** [comment]
hello,

This problem has been fixed since version [6.2.23b.1](https://github.com/sympa-community/sympa/releases/tag/6.2.23b.1).
If you want with your 6.2.22, you can customizeyour css.tt2 template with this hotfix:

> Fix CSS attributes for un-hovered navigation menu items using color_5 #94

Regards.

**Comment by racke** [comment]
And 6.2.24 is already available. Thanks for the hint, @aepli
